packages/compass-connections-navigation/src/index.ts

```diff
@@ -1,3 +1,8 @@
 export type { Actions } from './constants';
-export { type Connection } from './tree-data';
-export { NavigationWithPlaceholder as ConnectionsNavigationTree } from './connections-navigation-tree';
+export {
+  type Connection,
+  type ConnectedConnection,
+  type NotConnectedConnection,
+  type SidebarActionableItem,
```

Comment:
Should we export it as `SidebarItem`?

Reply:
makes sense, the container doesn't care about placeholders

Reply:
addressed this in [510f0b7](https://github.com/mongodb-js/compass/pull/5886/commits/510f0b77a22bf46964f180222753d6540d9c1532)